Upstream-Status: Backport

Signed-off-by: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
---
From 984335951bbc9b2d9532db8b8d1161494f79a105 Mon Sep 17 00:00:00 2001
From: "raphael.kubo.da.costa" <raphael.kubo.da.costa@intel.com>
Date: Wed, 17 Aug 2016 13:59:38 -0700
Subject: [PATCH] gn: Stop making |use_atk| depend on |use_gconf|.

This partly reverts commit 2a395c69f ("make use of existing gn args in
ui build config").

atk itself does not depend on gconf, and the change introduced in the
commit above was likely created before commit 95ba4446 ("Move linux
pkg_config() calls into separate BUILD.gn files"), when atk and gconf
were part of the same if block in build/config/linux/BUILD.gn.

See also: https://codereview.chromium.org/2241513002

R=brettw@chromium.org,dpranke@chromium.org,mostynb@opera.com
BUG=632297

Review-Url: https://codereview.chromium.org/2248043002
Cr-Commit-Position: refs/heads/master@{#412634}
---
 build/config/ui.gni | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/build/config/ui.gni b/build/config/ui.gni
index 849b720..1212ad2 100644
--- a/build/config/ui.gni
+++ b/build/config/ui.gni
@@ -18,7 +18,6 @@
 # of //build/config/BUILDCONFIG.gn.
 
 import("//build/config/chromecast_build.gni")
-import("//build/config/features.gni")
 
 declare_args() {
   # Indicates if Ash is enabled. Ash is the Aura Shell which provides a
@@ -87,7 +86,7 @@ if (is_linux && !use_ozone) {
 }
 
 # Whether to use atk, the Accessibility ToolKit library
-use_atk = is_desktop_linux && use_x11 && use_gconf
+use_atk = is_desktop_linux && use_x11
 
 use_clipboard_aurax11 =
     (is_desktop_linux && use_aura && use_x11) || enable_clipboard_aurax11
-- 
2.7.4

